src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/controller/ServiceClientHelper.java

```diff
@@ -104,4 +106,27 @@ public BadRequestException wrapInBadRequestException(
         return new BadRequestException(INVALID_SERVICE_CLIENT_ID + e.getServiceClientIdentifier(),
                 new ErrorDeviation(ERROR_INVALID_SERVICE_CLIENT_ID));
     }
+
+    /**
+     * Sort a list of ServiceClient objects using member name / group description as the primary sort key, and client id
+     * as the secondary sort key.
+     * @param clients
+     */
+    public void sortServiceClientsList(List<ServiceClient> clients) {
+        clients.sort(new Comparator<ServiceClient>() {
+            @Override
+            public int compare(ServiceClient c1, ServiceClient c2) {
+                if (c1.getName() == null) {
```

Comment:
Added null checks since it's possible that in some circumstances a client's name is not available, e.g., global conf has expired, client has been removed from global conf.